File: core/src/main/java/io/tcc/core/service/dto/RequestLabDTO.java
package io.tcc.core.service.dto;

import io.tcc.documentcommons.model.DocumentDTO;
import lombok.Getter;
import lombok.Setter;

import java.util.ArrayList;
import java.util.List;

@Getter
@Setter
public class RequestLabDTO {
    private String name;
    private String cpf;
    private String email;
    private String phone;
    private String title;
    private String description;
    private String resources;
    private String relation;
    private List<DocumentDTO> files = new ArrayList<>();
}

File: front/src/model/RequestLabDTO.ts
import {DocumentSaveDTO} from "./DocumentSaveDTO";

export class RequestLabDTO {
  description?: string;
  name?: string;
  cpf?: string;
  email?: string;
  phone?: string;
  relation?: string;
  title?: string;
  resources?: string;
  files?: DocumentSaveDTO[];
}
